src/App.js

```diff
@@ -9,16 +9,27 @@ import RegisterPage from './pages/RegisterPage';
 import DashboardPage from './pages/DashboardPage';
 
 export default function App() {
-  const [isAuthenticated, setIsAuthenticated] = useState(true); 
+  const [isAuthenticated, setIsAuthenticated] = useState(false); 
 
   const setAuth = (valueToSet)=>{
     setIsAuthenticated(valueToSet);
   }
 
   const isAuth = async()=>{
     try {
-        //TODO
-        // verify is user is authenticated and set state by that
+       
+      const response = await fetch('http://localhost:5000/auth/verify', {
```

Comment:
Okay this will not be a problem for you know but imagine when this is deployed, will it still be localhost? Or let's say it is on localhost, what happens if you change your `process.env.PORT` on backend and it is not `5000`

Reply:
Yes there is problem, will fix.